### PR TITLE
Add transpiled file and tooling

### LINKED ISSUE #47: Transpile JS
As far as I can see, the JS code is not transpiled and uses some rather new features / syntax (`async`, `const`...). 

Since we're launching in production, we should probably lower the bar for compatible browsers (IE11+), and thus add a transpilation step (or write ES5).

### PR DESCRIPTION
**Pull request by abulte**
Fix #47 #51

### CODE REVIEW

**Comment by abulte** [comment]
Sorry for the spam 😅 

Now that we have a JS package we could include the vendor dependencies (lunr...) through npm I guess, but that will be for another PR.

File: README.md [hunk 1]
```diff
@@ -27,6 +27,16 @@ document.head.appendChild(script)
 
 La zone de recherche s’ouvre au _focus_ pour proposer une version étendue, focalisant sur les résultats.
 
+#### Transpiler le code du widget
+
+Après avoir édité `js/src/widget.js` :
+
+```
+npm install
+npx babel js/src/widget.js --out-file js/widget.js
```
Comment (vinyll) [inline README.md:36]: Om mettrait pas un `npm run build` pour ce raccourci ?

File: package.json [hunk 2]
```diff
@@ -4,7 +4,8 @@
   "description": "## Intentions",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "npx babel js/src/widget.js --out-file js/widget.js"
```
Comment (vinyll) [inline package.json:8]: You should use directly `babel js…` here, no need of `npx` when it's called from package.json (which makes it compatible with previous versions).
Unless you have a difference info about that use case?

Comment (abulte) [reply package.json]: You're right 👌

**Pull request merged**